packages/dev/buildTools/src/webpackTools.ts

```diff
@@ -119,7 +120,9 @@ export const getRules = (
                         loader: "css-loader",
                         options: {
                             sourceMap: true,
-                            modules: true,
+                            modules: {
+                                localIdentName: options.mode === "development" ? "[path][name]__[local]" : "[hash:base64]",

```

Comment:
This considers prod being the default (if nothing is passed in mode) whereas the mode a bit later in the files default to dev :-)

Reply:
Goood catch Seb!